js/jquery.scripts.js

```diff
@@ -5,4 +5,6 @@ $( document ).ready(function() {
 	var result = test.replace(/\:/g,':<hr />');      
 
 	$('.post-title').html(result);
+
+	$('.recap').prepend("<h2>What did you learn?</h2>");
```

Comment:
The problem is that you can't seem to wrap a segment inside a <div> and keep using markdown inside it.

I wanted to create a recap segment with a different background so it stands out. I tried wrapping it into div and adding a "## What did you learn?" headline. Didn't work, so I opted out for inserting this H2 with jQuery.

Is there a better solution?